torch/tensor.py

```diff
@@ -183,7 +183,7 @@ def trim(str):
       also update the original tensor. Now, these in-place changes will not update the
       original tensor anymore, and will instead trigger an error.
       For sparse tensors:
-      in-place indices / values changes (such as `zero_` / `copy_` / `add_`) to the
+      In-place indices / values changes (such as `zero_` / `copy_` / `add_`) to the
       returned tensor will not update the original tensor either, and will instead
```

Comment:
Nit: write what you have above -- that it "will not update the original tensor anymore" rather than "will not update the original tensor either".  You want to make clear what is changed behavior.